cloud-run/clean-up-cloud-run.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ -z "$PROJECT" ]; then
+        echo "No PROJECT variable set"
+        exit
+fi
+
+if [ -z "$APIGEE_ENV" ]; then
+        echo "No APIGEE_ENV variable set"
+        exit
+fi
+
+if [ -z "$APIGEE_HOST" ]; then
```

Comment:
similar to checking if PROJECT, APIGEE_ENV and APIGEE_HOST are set, can we also include checks for CLOUD_RUN_SERVICE and CLOUD_RUN_REGION